compiler/include/bison-chapel.h

```diff
@@ -1,19 +1,19 @@
-/* A Bison parser, made by GNU Bison 3.0.4.  */
+/* A Bison parser, made by GNU Bison 2.5.  */
```

Comment:
Next time you make a change that involves rebuilding the parser, put the parser rebuild in a separate commit on its own. That makes ignoring it easier when reviewing.